src/gp/src/GPMSA.C

```diff
@@ -776,7 +778,8 @@ GPMSAFactory<V, M>::setUpEmulator()
   for (unsigned int i=0; i != m_numSimulations; ++i)
     for (unsigned int j=0; j != numOutputs; ++j)
       simulation_matrix(i,j) =
-        (*m_simulationOutputs[i])[j] - (*simulationOutputMeans)[j];
+        this->m_opts->normalized_output(j, (*m_simulationOutputs[i])[j]) -
+        (*simulationOutputMeans)[j];
```

Comment:
Verify that this isn't doubly applying the mean shift between the code here and in normalized_output.  (And in all places normalized_output is being called below.)